examples/safer_cluster_iap_bastion/variables.tf

```diff
@@ -69,3 +69,15 @@ variable "bastion_members" {
   description = "List of users, groups, SAs who need access to the bastion host"
   default     = []
 }
+
+variable "keyring" {
+  description = "Keyring name."
+  type        = string
+  default     = "gke-keyring"
+}
+
+variable "keys" {
+  description = "Key names."
+  type        = list(string)
+  default     = ["gke-key"]
+}
```

Comment:
We are transitioning to inline these variables so you can directly provide them in kms.tf

Reply:
@bharathkkb updated the comment